tests/ecdhp256-test.c

```diff
@@ -89,8 +93,134 @@ int main()
 	ok = ok && compare_and_print(64, point_compressed, compressed7);
 
 
+	printf("%s\n", "Wycheproof tests ECDH: ");
+		
+	for (int i = 0 ; i< sizeof(w_vectors)/sizeof(ecdhp256_w_i); i++)
+	{
+		printf("ECDH Wycheproof Test %d ", i );
+
+		uint8_t* decompressedPoint = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+		bool flagDecompress = Hacl_P256_decompression_not_compressed_form(w_vectors[i].publicKey, decompressedPoint);
+
+		uint64_t success = Hacl_P256_ecp256dh_r(result, decompressedPoint, w_vectors[i].privateKey);
```

Comment:
let's keep in mind that C tests are where our C users are looking for inspiration when it comes to how to use our API, so I would like to set the bar pretty high

why is this `success` not chained (via `&&`) with the result of the previous call to Hacl_P256_decompression_not_compressed_form?